### PR TITLE
Ahehn/warning fixes

### PR DESCRIPTION
**Pull request by ahehn-nv**
Fixed a couple of extra warnings.

### CODE REVIEW

**Comment by ahehn-nv** [comment]
Needed a relaxing task to get my brain working again. ;)

File: cudaaligner/src/matrix_cpu.hpp [hunk 1]
```diff
@@ -34,7 +34,7 @@ class matrix
 {
 public:
     using value_type = T;
-    static inline bool error(int t)
+    static inline bool error(int32_t t)
```
Comment (armintoepfer) [inline cudaaligner/src/matrix_cpu.hpp:37]: If you use fixed width integers, we usually also include `<cstdint>` then

Comment (ahehn-nv) [reply cudaaligner/src/matrix_cpu.hpp:37]: Good point.

Comment (ahehn-nv) [reply cudaaligner/src/matrix_cpu.hpp:38]: fixed.

**Review by mimaric — APPROVED** [review]

**Review by EpiSlim — APPROVED** [review]

**Comment by ahehn-nv** [comment]
rerun tests

**Pull request merged**